src/core/Akka/Actor/ActorRef.cs

```diff
@@ -68,38 +68,23 @@ public interface IRepointableRef : IActorRefScope
     public class FutureActorRef : MinimalActorRef
     {
         private readonly TaskCompletionSource<object> _result;
-        private readonly bool _tcsWasCreatedWithRunContinuationsAsynchronouslyAvailable;
         private readonly Action _unregister;
         private readonly ActorPath _path;
 
         /// <summary>
         /// TBD
         /// </summary>
         /// <param name="result">TBD</param>
-        /// <param name="unregister">TBD</param>
         /// <param name="path">TBD</param>
-        public FutureActorRef(TaskCompletionSource<object> result, Action unregister, ActorPath path)
```

Comment:
Need to revert these changes - this will break binary compatibility with prior versions of Akka.NET. Inside the JVM they use the `PromiseActorRef` instead of `FutureActorRef` here - the former being a replacement for the latter. We've started using `PromiseActorRef` in areas like graceful shutdown.